testmenu.c

```diff
@@ -143,28 +195,46 @@ int main(int argc, char **argv) {
 
                 rc = sqlite3_exec(db, sqltest, callback, 0, &zErrMsg);
 
-                if( rc != SQLITE_OK ) {
+                if( rc != SQLITE_OK )
+                {
                     fprintf(stderr, "SQL error: %s\n", zErrMsg);
                     sqlite3_free(zErrMsg);
-                } else {
+                } else
+                {
                     fprintf(stdout, "User created successfully\n");
+                    printf("You are now connected !\n");
+                    printf("Welcome %s ^^\n\n", username);
                     connected = 1;
                 }
 
-            }else if(choice == 3) {
+            }else if(choice == 9)
+            {
                 printf("Goodbye!\n");
                 exit(0);
             }
         // Case 2: User is connected
-        }else if(connected == 1) {
+        }else if(connected == 1)
+        {
             printf("Now, please choose an option:\n");
             printf("1. Add an other glycemia log\n");
             printf("2. See your glycemia logs\n");
             printf("3. See your glycemia logs for a specific date\n");
             printf("4. Log out\n");
+            printf("9. Exit\n");
             scanf("%d", &choice);
+
+            if(choice == 4)
+            {
+                printf("\nYou are now disconnected\n\n");
+                printf("Goodbye %s !\n\n", username);
+                connected = 0;
+            }else if(choice == 9)
+            {
+                printf("Goodbye!\n");
+                exit(0);
+            }
         }
-    }while(choice != 3);
+    }while(choice != 9);
```

Comment:
why 9

Reply:
to have the same number on both menu (connected and not) and like both does not have the same lenght it's easier to have the same number but can be change if it really doesn't make sense